### PR TITLE
Low hanging fruit refactors

### PR DESCRIPTION
**Pull request by JosephCurley**

### CODE REVIEW

File: src/components/saved-object/index.jsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const SavedObject = ({
 );
 
 SavedObject.propTypes = {
-	objectNumber: PropTypes.number,
+	objectNumber: PropTypes.string,
```
Comment (stephenhmarsh) [inline src/components/saved-object/index.jsx:31]: 150 emoji

File: src/components/active-object/index.jsx [hunk 2]
```diff
@@ -120,7 +120,8 @@ const ActiveObject = ({ object, handleSavedObjectChange, savedObjects }) => (
 					<a
 						target="_blank"
 						className="active-object__link"
-						href={object.primaryImage}>
+						href={object.primaryImage}
+						rel="noreferrer">
```
Comment (stephenhmarsh) [inline src/components/active-object/index.jsx:124]: curious why you added noreferrer here?

**Review by stephenhmarsh — APPROVED** [review]
looks good but see question thx

File: src/components/active-object/index.jsx (hunk 2 above)
Comment (JosephCurley) [reply src/components/active-object/index.jsx:124]: https://web.dev/external-anchors-use-rel-noopener/

Just something the linter made me aware of.

**Pull request merged**